### PR TITLE
Initial likes setup

### PR DESCRIPTION
**Pull request by samhstn**
See #70

### CODE REVIEW

**Comment by shouston3** [comment]
I would like to get the coverage up and sort out all the testing, but would prefer to do that in a separate pr as I would like to fix other things unrelated to this pr when doing so

**Review by Danwhy — APPROVED** [review]
Looking good :+1:

File: web/web.ex [hunk 1]
```diff
@@ -37,6 +37,7 @@ defmodule App.Web do
       import App.Router.Helpers
       import App.Gettext
       import App.Auth, only: [authenticate_user: 2]
+      require IEx
```
Comment (nelsonic) [inline web/web.ex]: are we keeping `IEx` in when this is in Production? 🤔

**Review by nelsonic — APPROVED** [review]
@shouston3 looks good. curious to get some detail on using `IEx` ... https://github.com/dwyl/learn-phoenix-framework/issues

File: priv/repo/migrations/20170509093107_add_likes_table.exs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+defmodule App.Repo.Migrations.AddLikesTable do
+  use Ecto.Migration
+
+  def change do
+    create table :likes do
+      add :user_hash, :string
+      add :article_id, :integer
+      add :like_value, :integer
```
Comment (SimonLab) [inline priv/repo/migrations/20170509093107_add_likes_table.exs]: Just curious why you choose to represent the likes with integer? My first thought is to use Boolean: true -> like; false -> dislike; nil -> no value

**Review by SimonLab — COMMENTED** [review]
@shouston3 the code looks good, just one question about the representation of likes in the database (integer vs boolean) and the comment from @nelsonic. Let me know your thoughts otherwise happy for this PR to be merged

File: web/web.ex (hunk 1 above)
Comment (shouston3) [inline web/web.ex]: Added ["adding debugging section"](https://github.com/dwyl/learn-phoenix-framework/issues/75) issue

Comment (shouston3) [inline web/web.ex]: I couldn't think of a way to pass IEx down to my controllers only in `test`/`dev` environments

File: priv/repo/migrations/20170509093107_add_likes_table.exs (hunk 2 above)
Comment (shouston3) [inline priv/repo/migrations/20170509093107_add_likes_table.exs]: I followed what @nelsonic wrote for me on storing these. I think either option is pretty clear

**Review by SimonLab — APPROVED** [review]
LGTM

**Pull request merged**